spec/10-overview.md

```diff
@@ -1,16 +1,13 @@
+Trace context is split into two individual propagation fields supporting interoperability and vendor-specifc extensability:
 
-The situation when a single request is a part of multiple graphs is becoming more common. One distributed application may have components monitored by different distributed tracing systems. Those distributed tracing systems may not be easily replaceable as they might be provided by cloud vendors or distributed as pre-built components.
+- `traceparent` describes the position of the incoming request in its trace graph in a portable, fixed-length format. Its design focuses on fast parsing. Every tracing tool MUST properly set `tracestate` even when it only relies on vendor-specif informaiton in `tracestate`
+- `tracestate` extends `traceparent` with vendor-specific data represented by a set of name/value pairs. Storing information in `tracestate` is optional.
 
-The trace context is designed to allow extensibility for all distributed tracing systems. It requires them to respect context set by other systems.
+Tracing tools can provide two levels of compliant behaviour interacting with trace context:
 
-Trace context is represented by a set of name/value pairs describing the identity of every http request. Two propagation fields carry the common and vendor-specific properties that make up the trace context.
+- At a minimum they MUST propagate the `traceparent` and `tracestate` headers and guarantee  traces are not broken. this behaviour is also referred to as forwarding a trace.
+- In addition they CAN also choose to participat in a trace by modifying the `traceparent` header and relevant parts of the `tracestate` header containing their proprietray information. This is also referred to as participating in a trace.
 
-* `traceparent` describes the position of the incoming request in its trace graph in a portable, fixed-length format. Its design focuses on fast parsing.
-* `tracestate` maps all graphs the incoming parent is a part of in potentially vendor-specific formats. For example, if a request crosses tracing systems, there will be one entry in `tracestate` for each system.
-
-Notably, the `tracestate` field is unreliant on data in the `traceparent`.
-
-Libraries and platforms MUST propagate `traceparent` and `tracestate` headers to guarantee that trace will not be broken.
+A tracing tool can choose to change this behavior for each individual request to a component it is monitoring. The decision on whether to participate or forward a trace is called sampling. A request is considered when the tracing tool participates in the trace.

```

Comment:
> A request is considered when …

Something missing here. Word "sampled"?

Reply:
Fixed in https://github.com/w3c/trace-context/pull/259/commits/1cb67acf85dc660d9daa69670785b53262e03ad3